Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_apache.sh
+# Description:
+#    Install apache and run apache performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+
+. ${CONSTANTS_FILE} || {
+	errMsg="Error: missing ${CONSTANTS_FILE} file"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 10
+}
+. ${UTIL_FILE} || {
+	errMsg="Error: missing ${UTIL_FILE} file"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 10
+}
+
+if [ ! "${server}" ]; then
+	errMsg="Please add/provide value for server in constants.sh. server=<server ip>"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 1
+fi
+if [ ! "${client}" ]; then
+	errMsg="Please add/provide value for client in constants.sh. client=<client ip>"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 1
+fi
+
+if [ ! "${nicName}" ]; then
+	errMsg="Please add/provide value for nicName in constants.sh. nicName=eth0/bond0"
+	LogErr "${errMsg}"
+	SetTestStateAborted
+	exit 1
+fi
+
+localaddress=$(hostname -i)
```

Comment:
What if this command returns error or "unknown host" ?

Reply:
This is redundant. I have deleted it.